bouncerscript/constants.py

```diff
@@ -101,4 +108,4 @@
         'https://bouncer-bouncer-releng.stage.mozaws.net/?product={}&print=yes',
 }
 
-NIGHTLY_VERSION_REGEX = r'\d+\.0a1'
+NIGHTLY_VERSION_REGEX = r'\d+\.\da1'
```

Comment:
@JohanLorenzo For the future, I believe we could/should import this thing directly from `mozilla-version`. I feel like we're duplicating some effort here. What do you think?

Reply:
I really liked this approach and I gave it a try but it fails under [these tests](https://github.com/mozilla-releng/bouncerscript/blob/master/bouncerscript/test/test_script.py#L123) (the main function which calls this with actual contents that bouncer is expecting, not just `:lang`). We can probably rewrite some of those tests but for now, I went ahead and adjusted the existing regex. But I really like this approach, much cleaner than before. Not to mention it gets to reuse `version_map` 👍 

I filed https://github.com/mozilla-releng/mozilla-version/issues/61 to track this in `mozilla-version`.